form.yml

```diff
@@ -0,0 +1,341 @@
+netweaver:
+  $name: Netweaver
+  $type: group
+  
+  install_packages:
+    $name: Install required packages
+    $type: boolean
+    $default: true
+    $help: Install all the required packages to install SAP Netweaver
+  
+  virtual_addresses:
+    $name: Virtual Addresses
+    $type: edit-group
+    $help: Define the IP addresses and hostnames for machines used in SAP Netweaver installation. These machines will be part of the HA cluster
+    $prototype:
+      $key:
+          $type: text
+          $name: IP Address
+      $type: text
+      $name: Hostname
+
+  sidadm_user_checkbox:
+    $name: Create the SAP sidadm user and sapsys group before the installation
+    $type: boolean
+    $help: Choose this option if the SAP sidadm user and sapsys group need to be created before the installation
+    $default: false      
+  sidadm_user:
+    $name: SAP User/Group
+    $visibleIf: .sidadm_user_checkbox == true
+    $type: group
+    $help: If this entry exists the user and group will be created before the installation, not otherwise
```

Comment:
`If this entry exists the user and group will be created before the installation, not during the installation [default]`